test/test_metadata.py

```diff
@@ -141,6 +143,18 @@ def func(x): return x[1:]
         self.assertEqual(MULTI_VALUED_JOINER.join(map(func, self.multi1)), self.metadata.get("multi1"))
         self.assertEqual(list(map(func, self.multi1)), self.metadata.getall("multi1"))
 
+    def test_metadata_applyfunc_preserve_tags(self):
+        self.assertTrue(len(PRESERVED_TAGS) > 0)
+        m = Metadata()
+        m[PRESERVED_TAGS[0]] = 'value1'
+        m['not_preserved'] = 'value2'
+
+        def func(x): return x[1:]
+        m.apply_func(func)
+
+        self.assertEqual("value1", m[PRESERVED_TAGS[0]])
+        self.assertEqual("alue2", m['not_preserved'])
+
```

Comment:
I just noticed applyfunc is ignoring tags from `PRESERVED_TAGS` but not from `preserved_tags` option (https://github.com/metabrainz/picard/blob/2f1943717ecd59c2ccbcdbf6187945489c25c736/picard/ui/options/tags.py#L47)
Bug or feature?

Reply:
Not sure really, but I'd say it's a bug. The only other place where we use `PRESERVED_TAGS` currently is in `file.copy_metadata`, and there `PRESERVED_TAGS` and `config.setting["preserved_tags"]` are treated the same.

Reply:
I'll address this later, we need a common util function for that.